edge/cdk/extensions/prewarm/lambda/scheduler/scheduler.py

```
@@ -116,13 +143,38 @@ def lambda_handler(event, context):
     pop_region = event_body['region']
     current_time = datetime.datetime.utcnow().strftime("%Y%m%dT%H%M%SZ")
 
+    # new added parameter
+    region_type = event_body['region_type']
     if type(pop_region) == str:
-        if pop_region.lower() not in pop_map.keys():
+        if pop_region != ALL:
```

I am not quit understand this logic, if pop_region is not 'all', we should raise the error?

---

pop_region can only accept one value of String type,and it is "all"
if the pop_region is "all",we should change the pop_region values named "all" to real pop list by region_type
I have commit this comment to this code block